authenticators/cf_authenticator_test.go

```diff
@@ -155,6 +199,17 @@ var _ = Describe("CFAuthenticator", func() {
 			Expect(metadata).To(Equal(metadata))
 		})
 
+		It("logs the access to the container by the user", func() {
+			Eventually(logger).Should(gbytes.Say("test.cf-authenticate.app-access-success"))
+			for _, msg := range logger.Logs() {
+				if msg.Message == "test.cf-authenticate.app-access-success" {
+					Expect(msg.Data["app"]).To(Equal("1e051b88-a210-40b7-bcca-df645b24b634/1"))
+					Expect(msg.Data["username"]).To(Equal("admin"))
+					Expect(msg.Data["principal"]).To(Equal("36ba11ff-0f6a-4c50-ab34-6fbd286a643e"))
+				}
+			}
```

Comment:
unfortunately this will not work.   If we do that I need to app the Exact message that will come including all the "timestamps" which are right now real.   I could rewrite the whole suite of tests with a fake clock but that seems like overkill!

I would need something like this ...

{"timestamp":"1479281731.219236612","source":"test","message":"test.cf-authenticate.app-access-success","log_level":1,"data":{"app":"1e051b88-a210-40b7-bcca-df645b24b634/1","principal":"36ba11ff-0f6a-4c50-ab34-6fbd286a643e","session":"1","username":"admin"}}

To match which is quite long and needs to match exactly.  The Say does not use fuzzy logic to only match certain fields in the logger data.  Plus since I don't care about the other fields the loop makes more sense.